amgut/templates/participant_overview.html

```diff
@@ -18,30 +18,53 @@
         <table class="list" id="survey" width="100%">
 {% if participant_type == 'human' %}
             <tr>
-                <td>{% raw tl['COMPLETED_CONSENT'] %}</td>
+                <td style="text-align: left">{% raw tl['COMPLETED_CONSENT'] %}:</td>
                 <td><img src="{% raw media_locale['SITEBASE'] %}/static/img/icon-green-checkmark.png"/></td>
             </tr>
 {% end %}
             <tr>
-                <td>{% raw tl['COMPLETED_SURVEY'] %}
-{% if participant_type == 'human' %}
-                    <form id="edit_survey" name="edit_survey" method="post" action="{% raw media_locale['SITEBASE'] %}/authed/human_survey/">
-                        <input type="hidden" name="survey_id" value={{ survey_id}}>
-                        <input type="hidden" name="page_number" value=-1>
-                        <input type="button" name="submit_button" id="submit_button" value="edit" onclick="submitEdit()">
-                    </form>
-                    {% raw vioscreen_text %}
-{% elif participant_type == 'animal' %}
-                    <form id="edit_survey" name="edit_survey" method="get" action="{% raw media_locale['SITEBASE'] %}/authed/add_animal/">
-                        <input type="hidden" name="survey" value={{ survey_id}}>
-                        <input type="button" name="submit_button" id="submit_button" value="edit" onclick="submitEdit()">
-                    </form>
+                <td style="text-align: left">{% raw tl['COMPLETED_SURVEY'] %}:
+					<ol>
+{% for survey in internal_surveys %}
```

Comment:
that was my first thought, but there are too many information that are different between those two survey types such that I decided to have a more speaking if condition rather than one more cryptic loop that covers both cases.
Is that OK with you?